bundles/org.openhab.ui/web/src/pages/settings/health/health-overview.vue

```diff
@@ -94,6 +115,11 @@ export default {
           this.semanticsProblemCount = data.length || 0
         })
       }
+      if (this.$store.getters.apiEndpoint('persistence')) {
+        this.$oh.api.get('/rest/persistence/persistencehealth').then((data) => {
+          self.persistenceProblemsCount = data.length || 0
```

Comment:
Self shouldn’t be required in an arrow function, they don’t bind their own this.